content/docs/studio/rest-api.md

```diff
@@ -0,0 +1,60 @@
+# REST API
+
+The purpose of Studio REST API is to give programmatic access to information in
+Studio and executing actions in it.
+
+The API is hosted under the `/api` route on the Studio server:
+https://studio.iterative.ai/api or https://your-domain/api in case of
+[self-hosted Studio](/doc/studio/self-hosting/installation).
+
+To use API, you need to generate
+[Studio access token](/doc/studio/user-guide/account-management#studio-access-token).
+
+## Download model
+
+Get signed url to download the model binaries for a model from Model Registry.
+Requires the model to be stored with DVC with s3 or azure remote. Note, that you
+need to
+[set up remote cloud credentials](/doc/studio/user-guide/account-management#cloud-credentials)
+for Studio have rights to sign urls.
+
+```yaml
+Endpoint: api/model-registry/get-download-uris
+HTTP Method: GET
+```
+
+### Request
+
+| param   | desc          | type   | required | example value                      |
+| ------- | ------------- | ------ | -------- | ---------------------------------- |
+| repo    | Git repo URL  | string | true     | iterative/demo-bank-customer-churn |
+| name    | Model name    | string | true     | randomforest-model                 |
+| version | Model version | string | false    | v2.0.0                             |
```

Comment:
What happened to supporting `stage` as suggested in https://github.com/iterative/studio/pull/6338#issuecomment-1593674785? Do you think it's needed?

Reply:
It's not implemented yet. It's on phase I list here https://github.com/iterative/studio/issues/5177